src/autotype/CMakeLists.txt

```diff
@@ -13,5 +13,8 @@ if(UNIX AND NOT APPLE)
 endif()
 
 if(WITH_TESTS)
-  add_subdirectory(test)
-endif()
+  # autotype non supported on Windows right now
+  if(UNIX)
```

Comment:
I don't like that ALL of the tests are relegated to UNIX builds only. We should move tests that are UNIX only (ie X11 autotype tests) to a subdir (tests/unix). WIN tests should be under tests/win.